src/OnboardingSPA/steps/SiteGen/Editor/Header/index.js

```diff
@@ -222,6 +222,26 @@ const StepSiteGenEditorHeader = () => {
 		}
 	}, [ currentData ] );
 
+	useEffect( () => {
+		const editorElement = document.querySelector(
+			'.nfd-interface-interface-skeleton__editor'
+		);
+
+		if ( editorElement ) {
+			if ( isSaving ) {
+				editorElement.classList.add( 'editor-actions-disabled' );
+			} else {
+				editorElement.classList.remove( 'editor-actions-disabled' );
+			}
+		}
+
+		return () => {
+			if ( editorElement ) {
+				editorElement.classList.remove( 'editor-actions-disabled' );
+			}
+		};
+	}, [ isSaving ] );
+
```

Comment:
We should create a state, something like `interactionDisabled`, and disable them at the parent level component using a `useEffect` instead of selecting the element by a class name, which can change with any WordPress release. We can make this a generic overlay state that can be used across any step in onboarding to prevent interaction. Let's also think about what we need to do incase we need to switch to an error state when something fails.